guides/IMPORT_EXISTING_DOCUMENTS.md

```diff
@@ -0,0 +1,67 @@
+The `firestore-meilisearch` script is for use with the official Firebase Extension [**Meilisearch**](https://github.com/firebase/extensions/tree/main/firestore-meilisearch).
+
+### Overview
+
+The import script can read all existing documents in a Cloud Firestore collection or in sub-collections and index them into an index in Meilisearch.
+
+#### Important notes
+
+- You must run the import script over the entire collection **_after_** installing the Meilisearch extension; otherwise the writes to your database during the import might not be exported to the your `${param:MEILISEARCH_INDEX_NAME}` index.
```

Comment:
> otherwise the writes to your database during the import might not be exported to the your `${param:MEILISEARCH_INDEX_NAME}` index.

This is a bit of a confusing sentence :/

Reply:
I change it tell me if it's better.